packages/cli/src/common/processor.ts

```diff
@@ -194,18 +193,9 @@ export async function processProject(
 }
 
 export function getUrl(hashes: string[], command: string) {
-  let url;
-  if (hashes.length === 1) {
-    url = new URL(
-      THIRDWEB_URL +
-        `/contracts/${command}/` +
-        encodeURIComponent(hashes[0].replace("ipfs://", "")),
-    );
-  } else {
-    url = new URL(THIRDWEB_URL + "/contracts/" + command);
-    for (let hash of hashes) {
-      url.searchParams.append("ipfs", hash.replace("ipfs://", ""));
-    }
+  const url = new URL(THIRDWEB_URL + "/contracts/" + command);
+  for (let hash of hashes) {
+    url.searchParams.append("ipfs", hash.replace("ipfs://", ""));
```

Comment:
does the dashboard handle single contract deploy/releases if i pass just one `ipfs` query param?